models/Element/Service.php

```diff
@@ -951,7 +958,7 @@ public static function addTreeFilterJoins(array $cv, Asset\Listing|DataObject\Li
 
                         $condition = $joinConfig['condition'];
                         $columns = $joinConfig['columns'];
-                        $select->add('select', $columns, true);
+                        $select->addSelect($columns);
```

Comment:
TODO: need to make sure $columns are strings, because `addSelect` accept only variantic string